src/lib/Store.ts

```diff
@@ -340,7 +191,80 @@ export class Store<T extends Piece> extends Collection<string, T> {
 			// Specifically ignore ENOENT, which is commonly raised by fs operations
 			// to indicate that a component of the specified pathname does not exist.
 			// No entity (file or directory) could be found by the given path.
-			if (error.code !== 'ENOENT') this.onError(error, path);
+			if (error.code !== 'ENOENT') this.strategy.onError(error, path);
 		}
 	}
+
+	/**
+	 * The injected variables that will be accessible to all stores and pieces. To add an extra property, simply mutate
+	 * the object to add it, and this will update all stores and pieces simultaneously.
+	 *
+	 * @example
+	 * ```typescript
+	 * // Add a reference to the Client:
+	 * import { Store } from '(at)sapphire/pieces';
+	 *
+	 * export class SapphireClient extends Client {
+	 *   constructor(options) {
+	 *     super(options);
+	 *
+	 *     Store.injectedContext.client = this;
+	 *   }
+	 * }
+	 *
+	 * // Can be placed anywhere in a TypeScript file, for JavaScript projects,
+	 * // you can create an `augments.d.ts` and place the code there.
+	 * declare module '(at)sapphire/pieces' {
```

Comment:
```suggestion
	 * declare module '@sapphire/pieces' {
```
whats up with not using @ now

Reply:
/resolved see previous comment